src/en/guide/how-to/cors.md

```diff
@@ -53,11 +53,7 @@ def _add_cors_headers(response, methods: Iterable[str]) -> None:
 
 def add_cors_headers(request, response):
     if request.method != "OPTIONS":
-        methods = [
-            method
-            for methods in request.route.methods.values()
-            for method in methods
-        ]
+        methods = [method for method in request.route.methods]
```

Comment:
We should test sanic versions so we can support 21.3 as well.

Reply:
may be the guide should have version too ,  more version check made the code doesn't clean

Reply:
You can ignore me. I read the PR and thought this was the sanic-ext  code. 😆 Ignore my comment. No versioning needed. The `en` user guide is meant to be current release, always. API docs get versioning. I added that to the sidebar.